gradle.properties

```diff
@@ -3,3 +3,5 @@ org.gradle.parallel=true
 jackson.version=2.9.2
 
 includePlatforms=ALL
+
+cloudProviders=ALL
```

Comment:
`includedPlatforms` and `cloudProviders` have the same intent - to only include specific modules when building the code. Would you be opposed to combining them to the same variable?

Reply:
Changed it to only `includCloudProviders`. Unfortunately, we have to include all projects till we can detangle the tests in clouddriver-web